### PR TITLE
Feauture Adding Linelist_Appointments

### PR DESCRIPTION
**Pull request by Marymary-dev**

### CODE REVIEW

File: Scripts/REPORTING/load_linelist_appointments [hunk 1]
```diff
@@ -0,0 +1,26 @@
+IF OBJECT_ID(N'[REPORTING].[dbo].LinelistAppointments', N'U') IS NOT NULL 		
+	drop table [REPORTING].[dbo].LinelistAppointments
+GO
+
+
+select
```
Comment (nobert-mumo) [inline Scripts/REPORTING/load_linelist_appointments:6]: @Marymary-dev :
- Does it make sense to also add all the other patient identifiers e.g. PatientIDHash, NUPIHash
- I think for the Linelist one of the big thing the Partners requested was to have the column on no of days missed or came before. we can add that
-  Do you think we need to have the Linelist going all the way back or we can cut for the last 2 years or so ? This dataset can become very big. Not sure though

Comment (Marymary-dev) [reply Scripts/REPORTING/load_linelist_appointments:6]: Done . Oh oh forgot to filter the data with Asof!

Comment (Marymary-dev) [reply Scripts/REPORTING/load_linelist_appointments:6]: Done

Comment (Marymary-dev) [reply Scripts/REPORTING/load_linelist_appointments]: @nobert-mumo  the column lateby has been renamed

**Review by nobert-mumo — APPROVED** [review]
LGTM

**Review by cbrianbet — APPROVED** [review]

**Comment by nobert-mumo** [comment]
@Marymary-dev please rem to add the new table and columns to the Dictionaries

**Pull request merged**